styles/nitrotype/catppuccin.user.css

```diff
@@ -0,0 +1,967 @@
+/* ==UserStyle==
+@name Nitro Type Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/nitrotype
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/nitrotype
+@version 0.0.1
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/nitrotype/catppuccin.user.css
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Anitrotype
+@description Soothing pastel theme for Nitro Type
+@author Ryan Wilson
+@license AGPL-3.0-or-later
```

Comment:
Is there a reason for changing these from Catppuccin and MIT

Reply:
It is a personal preference, but I will change it for the sake of conformity

Reply:
This entire repository is under MIT license, by contributing you give up your right to license the code.

Reply:
> This entire repository is under MIT license, by contributing you give up your right to license the code.

I have already changed the author field in newer commits